accepted/2025/dotnet-package-download.md

```diff
@@ -0,0 +1,121 @@
+# Dotnet Package Download Command
+
+* Author: [Nigusu](https://github.com/Nigusu-Allehu)
+* GitHub Issues: [#12513](https://github.com/NuGet/Home/issues/12513), [#14491](https://github.com/NuGet/Home/issues/14491)
+
+## Summary
+
+This document proposes adding a `dotnet package download` command to the .NET CLI.
+The command would enable package download to a local folder scenarios without requiring a project file.
+
+## Motivation
+
+Developers often need to download NuGet packages outside of a project context:
+
+* Populating internal or offline feeds.
+* CI/CD workflows that consume `.nupkg` artifacts directly.
+* **Cross-platform builds** where many customers today rely on `nuget.exe install` in CI scripts running on Linux or macOS, forcing them to install Mono just to get this functionality.
```

Comment:
`nuget install` is a bit sloppy with how it handles package source mapping.
My testing gave me
> Package 'Newtonsoft.json 13.0.4' is not found in the following primary source(s): 'https://api.nuget.org/v3/index.json,C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\'. Please verify all your online package sources are available (OR) package id, version are specified correctly.

At best, the error message isn't giving me a clear understanding of the problem (likely the logic finding the unmapped package is happening later than other commands like Restore). At worst, I wonder if install is trying to retrieve the package from the source first, then failing due to PSM being enabled.